### PR TITLE
New package: comfy-0.5.4

### PR DESCRIPTION
**Pull request by dacousb**
**General**
- [x] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

**Have the results of the proposed changes been tested?**
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me

### CODE REVIEW

File: srcpkgs/comfy/template [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'comfy'
+pkgname=comfy
+version=0.5.4
+revision=1
+build_style=cargo
+short_desc="Comfy is a flexible command script manager / runner written in Rust"
```
Comment (abenson) [inline srcpkgs/comfy/template:6]: Drop the "Comfy is a" and just start it like "Flexible command..."

Comment (dacousb) [reply srcpkgs/comfy/template]: Done. Thanks for the tip.

**Comment by kawaiiamber** [comment]
Squash the three commits to one - one commit per package. I recommend using `git commit --amend` for further commits after that.

File: srcpkgs/comfy/template [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'comfy'
+pkgname=comfy
+version=0.5.4
+revision=1
+build_style=cargo
+short_desc="Flexible command script manager / runner written in Rust"
+maintainer="daCoUSB <usbdaco@protonmail.com>"
+license="MIT"
+homepage="https://github.com/dacousb/comfy"
+distfiles="https://github.com/dacousb/comfy/archive/v${version}.tar.gz"
```
Comment (kawaiiamber) [inline srcpkgs/comfy/template:10]: Since the homepage is contained in distfiles, `distfiles="${homepage}/archive/v${version}.tar.gz"` might look better.

**Review by kawaiiamber — COMMENTED** [review]
Simplify `distfiles`

**Comment by kawaiiamber** [comment]
The four commits still need to be squashed to one to adhere to the one commit per package rule. `git rebase -i HEAD~4`, choose `squash` for the last three, and `git push origin master -f`. Personal recommendation, I might make a branch relating to the package `git checkout -b comfy` and PR from that branch, that's just me though.

**Comment by dacousb** [comment]
> The four commits still need to be squashed to one to adhere to the one commit per package rule. `git rebase -i HEAD~4`, choose `squash` for the last three, and `git push origin master -f`. Personal recommendation, I might make a branch relating to the package `git checkout -b comfy` and PR from that branch, that's just me though.

I've tried and I don't really know why, it does not work. It may be because I've done some changes with GitHub Web. Sorry!

**Comment by kawaiiamber** [comment]
No problem! It's an easy fix ^-^. I would highly recommend using the `git` command line utility as that is what I can help with. If you're using void yourself, `xbps-install -S git` if you haven't done so already. The first thing you want to do is clone your fork on `void-packages`. For https:
```
git clone https://github.com/dacousb/void-packages.git
cd void-packages
```
Add the void-linux `void-packages` repo as an upstream remote:
```
git remote add upstream https://github.com/void-linux/void-packages.git
```
I would copy the template you made for comfy somewhere, then checkout on a new branch:
```
git checkout -b comfy
git pull --rebase upstream master # not really needed if you JUST cloned it :)
```
Put the directory / template in for comfy, then add that directory to the tracking index: `git add srcpkgs/comfy`
Then put it to your fork:
```
git commit -m "New package: comfy-0.5.4"
git push origin comfy
```
Then, just click "edit" on the top right of this PR and chance from your msater branch to the "comfy" branch. For editing this PR, just use `--amend`:
```
# make your changes
git add srcpkgs/comfy
git commit --amend # write and quit with same commit message
git push origin comfy -f # -f to force push as your ammending a commit
```

OR

If you want to merge on this branch (master branch on your fork), here's a quick guide to squashing commits with `rebase`:
```
git rebase -i HEAD~5 # since there's five commits
```
This will take you into an interactive rebase (that's what the `-i` is for :3). There will be five lines at the top that say "pick <commit message>" and comments below telling you what the commands do. We want the first one to be "pick". To edit the last four, it's like vim. `i` to go into insert mode, then just replace the last four `pick` with `squash` instead. Then `Esc + :wq + retrun`. Then force push this to your master branch to squash the commits: `git push origin master -f`. Again, at this point, use `--amend` for future changes to the PR:
```
# make your changes
git add srcpkgs/comfy
git commit --amend # write with same commit msg
git push origin master -f
```

I hope this was helpful - let me know if you have questions, either here or email me (on my github profile) ^-^

**Comment by dacousb** [comment]
@kawaiiamber thanks for your support! anyway, every time I `clone` and `rebase` the repository, I get errors like:
image

**Comment by kawaiiamber** [comment]
> @kawaiiamber thanks for your support! anyway, every time I `clone` and `rebase` the repository, I get errors like:
> image

What's the full error / exact command?

**Comment by dacousb** [comment]
@kawaiiamber oh I'm so sorry, I totally forgot about this... Here it is the error:
image
Thanks in advance!

**Comment by kawaiiamber** [comment]
```
git stash
git reset -f
git add srcpkgs/comfy
```

**Comment by dacousb** [comment]
@kawaiiamber still does not work, I'm struggling so hard with this, other repos just work with the rebase... I don't know what to do.

**Pull request closed without merging**